docs/src/modules/components/DemoEditorHeader.js

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import PropTypes from 'prop-types';
+import { NoSsr } from '@mui/base';
+import { blue } from 'docs/src/modules/brandingTheme';
+import { useTranslate } from 'docs/src/modules/utils/i18n';
+import CodeCopyButton from './CodeCopyButton';
+
+const Root = styled('div')(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  height: 40,
+  paddingInline: theme.spacing(2),
+  borderBottom: `1px solid ${theme.palette.primaryDark[500]}`,
+  '& > .MuiCode-copy': {
+    display: 'block',
+    top: 'unset',
+    '& .MuiCode-copyKeypress': {
+      top: 0,
+      left: '-50%',
+    },
+  },
+}));
+
+const Text = styled(Typography)(({ theme }) => ({
+  textTransform: 'uppercase',
+  color: blue[100],
+  fontWeight: 500,
+  fontSize: theme.typography.pxToRem(10),
+}));
```

Comment:
```suggestion
const HeaderText = styled(Typography)(({ theme }) => ({
  fontSize: theme.typography.pxToRem(10),
  fontWeight: theme.typography.fontWeightBold,
  textTransform: 'uppercase',
  letterSpacing: '.08rem',
  color: theme.palette.primary[200],
}));
```

Fine-tunning the styles and changing the component name to a slightly more descriptive one.